collector/cmd/otelarrowcol/main.go

```diff
@@ -16,8 +17,8 @@ func main() {
 
 	info := component.BuildInfo{
 		Command:     "otelarrowcol",
-		Description: "Local OpenTelemetry Collector Arrow binary, testing only.",
-		Version:     "0.0.1-dev",
+		Description: "Development OTel-Arrow Collector binary, testing only.",
+		Version:     "1.0.0",
```

Comment:
I'm trying to figure out if it's logical that in versions.yaml the version number is 0.1.0 but in the source code collector/cmd/otelarrowcol/main.go it's 1.0.0

Reply:
Fixed, see https://github.com/open-telemetry/otel-arrow/pull/10/commits/3944bb43ffa3d34b139297920e75ea2945c43206